content/english/updates/Pathogen_user_survey_25.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Survey to improve the Swedish Pathogens Portal"
+date: 2025-09-08
+summary: The Swedish Pathogens Portal is seeking feedback from users to identify how it can be improved to better meet their needs. Please complete the survey to share your thoughts!
+---
+
+The Swedish Pathogens Portal is a community-driven portal. This means that it continuously changes and adapts in accordance with the needs of the community. In order to do this effectively, it is crucial to gain feedback and input from them.
```

Comment:
```suggestion
The Swedish Pathogens Portal is a community-driven portal. As such, it continuously adapts to best serve the needs of its community. To be able to do this effectively, we're seeking your feedback on what you need from the Swedish Pathogens Portal, and what we can do to improve it, so that it remains a valuable resource.```